node/nodeRunner.go

```diff
@@ -320,16 +320,34 @@ func (nr *nodeRunner) executeOneComponentCreationCycle(
 		return true, err
 	}
 
+	// The accounts adapter isn't yet available, it will be set a bit later (see below).
```

Comment:
State components are created at a later time.

Reply:
chicken and egg :D